astronomy/eclipse_test.cpp

```diff
@@ -250,28 +214,29 @@ class EclipseTest : public testing::Test {
         current_time - bisection_interval,
         current_time + bisection_interval);
     EXPECT_EQ(Sign(actual_contact_time - current_time),
-              Sign(time_error)) << NAMED(time_error);
+              Sign(time_error)) << NAMED(actual_contact_time - current_time);
```

Comment:
Why are we testing the sign of the difference and its absolute value separately anyway (especially since we end up remultiplying them for logging)?